types/react-native/index.d.ts

```diff
@@ -1545,7 +1546,8 @@ export interface ViewStyle extends FlexStyle, TransformsStyle {
     borderTopColor?: string;
     borderTopLeftRadius?: number;
     borderTopRightRadius?: number;
-    borderTopWidth?: number
+    borderTopWidth?: number;
+    color?: string;    
     opacity?: number;
```

Comment:
And could you give the source.
I cannot find `color` attribute in the following sources.

`https://github.com/facebook/react-native/blob/master/React/Views/RCTView.m
`
`https://github.com/facebook/react-native/blob/master/React/Views/RCTViewManager.m`